channel.go

```diff
@@ -786,6 +970,27 @@ can be used to test for the existence of a queue.
 
 */
 func (ch *Channel) QueueDeclarePassive(name string, durable, autoDelete, exclusive, noWait bool, args Table) (Queue, error) {
+	return ch.QueueDeclarePassiveContext(
+		context.Background(),
+		name,
+		durable,
+		autoDelete,
+		exclusive,
+		noWait,
+		args,
+	)
+}
+
+/*
+
+QueueDeclarePassiveContext is functionally and parametrically equivalent to
+QueueDeclareContext, except that it sets the "passive" attribute to true.
+A passive queue is assumed by RabbitMQ to already exist, and attempting to
+connect to a non-existent queue will cause RabbitMQ to throw an exception.
```

Comment:
Clients connect to nodes, not queues. "Attempting to passively declare a non-existent queue will result in a channel-level protocol exception".